packages/next/server/dev/hot-reloader.ts

```diff
@@ -361,6 +361,10 @@ export default class HotReloader {
               break
             }
             case 'client-full-reload': {
+              traceChild = {
+                name: payload.event,
+                attrs: { stackTrace: payload.stackTrace ?? '' },
```

Comment:
Does it make sense to add the stackTrace here?

Reply:
The additional context does seem nice to have here